app/src/main/res/values/strings.xml

```diff
@@ -61,7 +61,9 @@
     <string name="disc_number">"Disc Number"</string>
     <string name="lyrics">Lyrics</string>
     <string name="album_artist_multiple">Album Artists (one per line)</string>
+    <string name="album_genre_multiple">Album Genres (one per line)</string>
```

Comment:
We can drop this, album genre and (track) genre are the same, they all being saving in the same tag.

Reply:
Done